graftm/hmmer.py

```diff
@@ -229,37 +230,39 @@ def merge_forev_aln(self, forward_aln_list, reverse_aln_list, outputs):
         -------
         Nothing - output files are known.
         '''
-
+        slash_ending_regex = re.compile('.*/[12]$')
```

Comment:
Ah right I'd forgotten. Python uses the (I believe non-standard) 
approach that match has to apply to the beginning of the string, where 
another method (search?) does not.